### ISSUE TITLE
Abnormal wrist extension

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5907]
**Preferred term label:** Abnormal wrist extension

**Synonyms**

**Definition (free text, please give PubMed ID)** Limited extension or hyperextension of the wrist.

**Parent term (use hpo.jax.org/app)** Limited wrist movement HP:0006248

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
This could be the parent term for Limited wrist extension HP:0006251

**Comment by pnrobinson** [comment]
I think that this term is what we want for this -- https://hpo.jax.org/app/browse/term/HP:0006248
